### PR TITLE
Enable kube-rbac-proxy to be able to reach metrics

### LINKED ISSUE #196: [Question] OLM, exposing metrics
Hello!

I was able to install self-node-remediation via OLM and asking for help now.

I'd like to access metrics, but metrics endpoint listens to `127.0.0.1:8080` so they are unreachable for prometheus instance. This value is set with `args` in `Deployment`.

I could set `metrics-bind-address` to `:8080` with `Subscription` `spec.config`, but args value has the priorirty.

Is it on purpose?
What's the proper/proposed way to deal with it?

Thank you.

### PR DESCRIPTION
**Pull request by agrrh**
#### Why we need this PR
It enables kube-rbac-proxy container so we could reach metrics.

#### Changes made
- Uncomment `manager_auth_proxy_patch.yaml`
- Regenerate bundle manifests

#### Which issue(s) this PR fixes
Fixes #196, as I hope :)

[ECOPROJECT-1949](https://issues.redhat.com//browse/ECOPROJECT-1949)

### CODE REVIEW

File: bundle/manifests/self-node-remediation.clusterserviceversion.yaml
Comment (slintes) [inline bundle/manifests/self-node-remediation.clusterserviceversion.yaml:394]: Can you please update the patch file for setting `resources` and `securityContext` like here? https://github.com/medik8s/node-healthcheck-operator/blob/main/config/default/manager_auth_proxy_patch.yaml#L23-L34

**Review by slintes — CHANGES_REQUESTED** [review]
Thanks for the PR :) The patch file needs an update, see inline comment please.

**Comment by agrrh** [comment]
Seems like some random failure:

```
Self Node Remediation E2E Control Plane Remediation [AfterEach] With API connectivity creating a SNR Resource Deletion Strategy should delete pods and volume attachments
  [AfterEach] /go/src/github.com/medik8s/self-node-remediation/e2e/self_node_remediation_test.go:317
  [It] /go/src/github.com/medik8s/self-node-remediation/e2e/self_node_remediation_test.go:358
  [FAILED] Timed out after 1200.009s.
  pod did not get ready in time
  Expected
      <v1.ConditionStatus>: False
  to equal
      <v1.ConditionStatus>: True
  In [AfterEach] at: /go/src/github.com/medik8s/self-node-remediation/e2e/self_node_remediation_test.go:619 @ 04/24/24 11:35:50.151
```

Not sure if I'm allowed to trigger CI here. Would give it a try.

**Comment by agrrh** [comment]
Yep, I'm not allowed to restart it.

@slintes kindly asking you to take a look, please.

**Review by slintes — APPROVED** [review]
lgtm, thanks!
/ok-to-test

/hold
this needs an update in downstream CI before merging
@mshitrit fyi, please ~approve and~ unhold when ready

File: bundle/manifests/self-node-remediation.clusterserviceversion.yaml
Comment (agrrh) [reply bundle/manifests/self-node-remediation.clusterserviceversion.yaml]: Whoops

This is done, thank you!

**Pull request merged**
